src/lib/sql/sql_translator.cpp

```diff
@@ -527,8 +527,11 @@ SQLTranslator::TableSourceState SQLTranslator::_translate_table_origin(const hsq
         }
 
       } else if (Hyrise::get().storage_manager.has_table(hsql_table_ref.name)) {
-        lqp = _translate_stored_table(hsql_table_ref.name, sql_identifier_resolver);
-
+        if (MetaTableManager::is_meta_table_name(hsql_table_ref.name)) {
```

Comment:
Do we still need the StorageManager to return `has_table` or can we completely switch to the SQLTranslator handling this?